tests/integration/REST_API/Controllers/SettingsTest.php

```diff
@@ -38,46 +81,50 @@ protected function _after() {
 	/** Tests *********************************************************************************************************/
 
 
-	/** @see Settings::get_item() */
-	public function test_get_item() {
```

Comment:
Oh, it got moved to after the helper methods... I believe it would be better if it was right after `test_get_items`. And the helpers after all the test methods.

Reply:
Sounds good. I moved the helpers to the bottom in e0fabbe.